test/test-api/src/lib.rs

```diff
@@ -41,10 +41,6 @@ pub enum AssistOp {
 /// Operations that are performed by the test-suite
 #[derive(FromPrimitive)]
 pub enum SuiteOp {
-    /// Get the number of test cases (`() -> usize`).
-    GetCaseCount = 1,
-    /// Get the name of a case (`usize -> [u8]`).
-    GetCaseName = 2,
     /// Run a case, replying before it starts (`usize -> ()`).
     RunCase = 3,
```

Comment:
Is it worth just eliminating the `SuiteOp` altogether given that there is only one of them now?

Reply:
I thought about that but for consistency I like keeping the enum